src/tests.rs

```diff
@@ -0,0 +1,23 @@
+use crate::{mock::*};
+use frame_support::{assert_ok};
+
+#[test]
+fn issue_key() {
+    new_test_ext().execute_with(|| {
+        let mut fingerprint: Vec<u8> = Vec::new();
+        let mut location: Vec<u8> = Vec::new();
+        fingerprint.push(1);
+        location.push(1);
+        assert_ok!(KeystoreModule::issue_key(Origin::signed(1), fingerprint, location));
```

Comment:
I think you probably already knew this but I want to make sure the code was clear that the `Vec<u8>`s here are used to represent WASM-compatible strings. So you can take `"some string".to_vec()` and as long as the string is ascii-encodable it should store.